src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
```

Comment:
```suggestion
export default async function report(runningTests, startTime) {
```
?

Reply:
👍 
Since exporting and declaring are two different actions it felt clearer as two different statements.
I don't have a strong preference either way, and the inline export style you propose is common enough that I don't think it will cause any confusion so I'll go ahead and change it throughout the project :)